lecture_2/homework/app/controllers/concerns/error_handler.rb

```diff
@@ -0,0 +1,26 @@
+module ErrorHandler
```

Comment:
Nie za dużo się dzieje w tym module? Obsługę każdego z błędów masz napisaną dwa razy - tylko jedna z nich będzie działać 😃

Reply:
A wręcz jedna z nich spowoduje, że nasza aplikacja nie będzie działać w ogóle, ponieważ dostaniemy błąd `undefined method `rescue_from' for ErrorHandler:Module`